### PR TITLE
Added new endpoint to format cql

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-5066](https://jira.cms.gov/browse/MAT-5066)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by adongare — DISMISSED** [review]

File: src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlToolsController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class CqlToolsController {
+
+  @PutMapping("/cql/format")
+  public ResponseEntity<String> formatCql(@RequestBody String cqlData, Principal principal) {
+    var cqlDataStream = new ByteArrayInputStream(cqlData.getBytes());
+    try {
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlToolsController.java]: Since [ByteArrayInputStream](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/io/ByteArrayInputStream.html) implements the `AutoCloseable` interface, it can be used in a try-with-resources block to ensure it's closed after the try/catch block finishes execution. 

```suggestion
try (var cqlDataStream = new ByteArrayInputStream(cqlData.getBytes())) {
```

**Review by jkotanchik-SB — APPROVED** [review]
